apps/api/src/utils/user_record.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+from pydantic import Field
+
+from services.mongodb_handler import BaseRecord
+
+
+class Role(str, Enum):
```

Comment:
Remind me again, how did we assign mentor/volunteer roles last year? Sponsors and judges should also be able to view the schedule.

Reply:
I don't think we ended up using all of these roles last year, but the concept was mentors and volunteers could also log into the portal and see personalized information, e.g. about their shifts or other event information they might want to know. To actually assign roles, we would need to do it manually in the database or implement a user management interface in the admin portal.

For the schedule, we made it public last year (HackAtUCI/HackUCI-Site@55fa9a2), not sure if you want to handle that differently this year.

Reply:
Got it, yeah public schedule is fine.